### PR TITLE
[ADD] Added OCA Maintainer role Policy

### PR DESCRIPTION
**Pull request by elicoidal**
@max3903 
cc @OCA/board

### CODE REVIEW

File: website/Contribution/oca_module_lifecycle_maintainer_role.rst [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+under `Creative Commons Attribution 4.0 International License <http://creativecommons.org/licenses/by/4.0/>`_.
+
+Maintainer
+----------
+OCA, or the `Odoo Community Association <https://odoo-community.org>`_,
+is a nonprofit organization whose mission is to support the collaborative
+development of Odoo features and promote its widespread use.
+
+If you want to contribute, please check our
+`website <https://odoo-community.org/page/Contribute>`_.
```
Comment (sbidoul) [inline website/Contribution/oca_module_lifecycle_maintainer_role.rst]: For the sake of simplicity, could we consider the git repo provides this metadata (rev history, license, etc), and therefore remove it from the text?

Comment (elicoidal) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: @sbidoul We could be but we should consider that there are multiple output for this documentation: website, pdf etc. which in the end might be unlinked with the original document.
Each document might have different path.

For this reason I am not sure whether this is a good idea (not against: just not sure)

Comment (sbidoul) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: About publishing, what's the plan to publish these documents?
1. link to the github preview from the main OCA website
2. generate a standalone html page from each document
3. generate a sphinx website with it's own menu

1 and 2 will create issues to create/maintain links between documents.

3 has the drawback of being kind of a second website.

Thoughts?

Comment (sbidoul) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: @elicoidal @OCA/board I can setup travis but before I start something, what are your thoughts on my above question.

File: website/Contribution/oca_module_lifecycle_maintainer_role.rst [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+.. |CATEGORIES| replace:: Contribution, Policy, Maintainer Role
+
+.. fill in the Document information in the above lines. It will be automatically
+   replaced in the document header. Don't forget to remove the []
+
+.. contents:: Table of contents
+    :depth: 4
+
+.. sectnum::
+
```
Comment (sbidoul) [inline website/Contribution/oca_module_lifecycle_maintainer_role.rst]: @elicoidal Can you insert a word saying the canonical URL to this document is https://odoo-community.org/page/maintainer-role

I'll adapt the redirection when it's merged.

File: website/Contribution/oca_module_lifecycle_maintainer_role.rst (hunk 1 above)
Comment (elicoidal) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: @sbidoul I think the idea is right away 1 and later 2 with static pages on the current website.
for 1 and 2 indeed we will have to maintain hardcoded links between documents when necessary but IMO not many of them and most of these documents will not evolve much/often.
3 IMO is painful in terms of communication.

On my side I have not received a clear answer on the footer. 
cc @OCA/board

File: website/Contribution/oca_module_lifecycle_maintainer_role.rst (hunk 2 above)
Comment (elicoidal) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: done

**Comment by elicoidal** [comment]
@sbidoul please review again.

File: website/Contribution/oca_module_lifecycle_maintainer_role.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+-  Addons with no declared maintainers are managed by the PSC.
+
+Adding, removing maintainers
+============================
+
+-  The maintainers of an addon are declared in the **maintainers** key
+       of the manifest. This key is a list of github logins. The list of
+       declared maintainers is expected to be short (one or two in
+       general).
```
Comment (sbidoul) [inline website/Contribution/oca_module_lifecycle_maintainer_role.rst]: @elicoidal there are some rendering issues (see https://github.com/OCA/odoo-community.org/blob/6a293cd6d265a1383390a7360ed801a9e679898a/website/Contribution/oca_module_lifecycle_maintainer_role.rst). 

One needs to choose between bullet list and definition list.

Comment (elicoidal) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: right: I will have a look at it!

Comment (elicoidal) [reply website/Contribution/oca_module_lifecycle_maintainer_role.rst]: done

**Comment by elicoidal** [comment]
can we merge this? @OCA/board

**Pull request merged**